<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Theses | Sudharshan Ravi, Ph.D. </title> <meta name="author" content="Sudharshan Ravi, Ph.D."> <meta name="description" content="Computation Biologist at UZH, Zurich "> <meta name="keywords" content="computational biologist, metabolic network analysis, bioinformatics, systems biology"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/Logo2.png?03f154e9931b9a76e3a320234b809c84"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ravisudharsh.github.io/theses/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?757986ae90a8b8966263cb9d9a515daa"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Sudharshan</span> Ravi, Ph.D. </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">Research </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">Publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/supervision/">Supervision </a> </li> <li class="nav-item "> <a class="nav-link" href="/assets/pdf/CV_SRavi.pdf" target="_blank">Vitae </a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">Contact </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Theses</h1> <p class="post-description"></p> </header> <article> <div class="clearfix" style="text-align:justify"> <div class="publications"> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/eth_logo-480.webp 480w,/assets/img/publication_preview/eth_logo-800.webp 800w,/assets/img/publication_preview/eth_logo-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/eth_logo.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="eth_logo.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Ravi_phd" class="col-sm-8"> <div class="title">Metabolic network analysis and its application in understanding the biology of aging</div> <div class="author"> <em>Sudharshan Ravi</em> </div> <div class="periodical"> <em>ETH, Zürich</em> , 2020 </div> <div class="periodical"> Ph.D. Thesis </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/theses/doctoral_thesis.pdf" target="_blank" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://www.research-collection.ethz.ch/handle/20.500.11850/484454" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="abstract hidden"> <p>A fundamental aim of life science is to understand the functioning of an organism and draw system-level connections between its genotype and behavior. Among the most significant types of interaction networks, cellular metabolism is a well-established descriptor of the phenotype of an organism. It relies on thousands of enzymatic reactions that are often represented as a dense and enmeshed web of biochemical conversions. Inspecting the metabolic network in its entirety is crucial in attaining a holistic understanding of the underlying biological mechanism. Colossal research efforts in the post-genomic era have enabled the curation of metabolic networks of entire organisms. Concurrently, advancements in computational strategies and algorithms have led to the inception of countless tools that utilize the essential information in genome-scale metabolic models to attain valuable insights into the physiology of an organism. Constraint-based modeling is a widely used and tested technique to model the metabolic network. Flux balance analysis (FBA) is the most popular constraint-based approach to predict intracellular metabolic flux distributions and network capabilities in genome-scale models. They are modeled on two fundamental assumptions. Firstly, that the cellular metabolism operates at a homeostatic condition, and secondly, that the cell typically organizes its metabolism to optimize a particular cellular objective. However, these assumptions significantly impair the applicability of the traditional flux balance analysis. Where the prediction accuracy is insufficient, additional constraints from omics data sources assist in obtaining biologically relevant inference. In this work, we showcase the addition of transcriptomic data to gain a profound understanding of the early events in the progression of Alzheimer’s disease model Caenorhabditis elegans. Transgenically expressing human amyloid-beta recapitulated the phenotypic disease response in the worms. Our analyses of the contextualized genome-scale metabolic network curated with the integration of experimentally derived gene expression data implicated metabolic alterations in Tricarboxylic Acid (TCA) cycle following low-level amyloid-beta expression. Along with metabolomic and enzymatic assays, we show repression in alpha-Ketoglutarate dehydrogenase. Identification of metabolic dysfunction as an early event is paramount in formulating mitigating efforts. Formulating a metabolic objective for use in constraint-based modeling is hazy, particularly for complex multicellular organisms. In addressing such concerns, we developed a computational algorithm, ∆FBA (delta–FBA), that focuses on the differences in the metabolic distribution between a pair of conditions. By formulating the mathematical problem to optimize for maximal consistency between the inferred flux alterations and integrated gene expression changes, ∆FBA predicts metabolic rewiring as an effect of genetic or environmental perturbations. We validated our strategy in a wide range of single-gene deletion knockouts and environmental modifications in Escherichia coli, where ∆FBA outperforms similar methods. Furthermore, our findings of metabolic changes in human diabetic subjects show the robustness of ∆FBA. Akin to metabolism, aging is a complex biological process that necessitates a system-level analysis in unraveling its etiology and progression. Despite numerous efforts, the aging process in humans is far from being completely understood. The Genotype-Tissue Expression (GTEx) project represents an invaluable repository of information that is ideal for studying human aging by examining the differences in the gene expression of over seven hundred individuals. Our bioinformatics and metabolic network analysis of the transcriptome associated human aging with evolutionarily conserved hallmarks of aging. Additionally, we show that the temporal changes in gene expression significantly contribute to the aging process. Our findings suggest that persistent moderators of cell fate and early repressors of cellular energetics could play a pivotal role in the progression of the aging process, one that culminates in accelerated decline.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@thesis</span><span class="p">{</span><span class="nl">Ravi_phd</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ravi, Sudharshan}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Metabolic network analysis and its application in understanding the biology of aging}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{ETH, Z{\"u}rich}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3929/ethz-b-000484454}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.research-collection.ethz.ch/handle/20.500.11850/484454}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{Ph.D. Thesis}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/eth_logo-480.webp 480w,/assets/img/publication_preview/eth_logo-800.webp 800w,/assets/img/publication_preview/eth_logo-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/eth_logo.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="eth_logo.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Ravi_msc" class="col-sm-8"> <div class="title">Reducing Complexity — Efficient Modeling for Biorefinery Concepts</div> <div class="author"> <em>Sudharshan Ravi</em> </div> <div class="periodical"> <em>ETH, Zürich</em> , 2015 </div> <div class="periodical"> M.Sc. Thesis </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/theses/master_thesis.pdf" target="_blank" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>An innovative integrated biorefinery for the production of Polyethylene terephthalate (PET) and Polyurethane (PU) was developed with the purview of the ‘Waldbox’ project. PET is an extensively used polymer in the production of food and beverage containers and an essential component in the manufacturing of synthetic fibers. With a far-reaching impact on humanity, PU has found considerable use and emphasis in the modern-day world. Manufacture of high-resilience foam seating, rigid-foam insulation panels, durable elastomeric wheels and tires, and high performance adhesives are some of the inexhaustible applications of the versatile polymer. Commercially production of the two critical polymers stems through the crude oil reforming. The alternative process, presented within this master thesis, demonstrates the process design for bio-based production of PET and PU in an integrated biorefinery using lignocellulosic-based d-glucose as feedstock. The simulations were performed using Aspen Plus®. Environmental and economic assessment of the alternative process delineates its competitiveness compared to the commercial production of the individual polymers. The introduction of a new generation of biorefineries into the market presents a complex problem, where questions like environmental sustainability, economic feasibility and conceptual exploration need to be answered beforehand to find an optimal concept. Such an integration requires the use of reliable and robust optimization tools. Surrogate modeling offers an innovative approach for breaking down complex process design to integrate them into a computer aided platform. This minimalistic approach can lead to an optimized and integrated biorefinery concept regardless of the complexity of the design range. The method was applied to the innovative integrated biorefinery to predict the behavior of the models even outside the simulated range, thereby proving an important tool that can be used for translating complex process design into simplified and accurate models. This would be the first step in building a computer-aided platform for the integration of different bio-based production routes to find the optimal biorefinery concept.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@thesis</span><span class="p">{</span><span class="nl">Ravi_msc</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ravi, Sudharshan}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Reducing Complexity — Efficient Modeling for Biorefinery Concepts}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{ETH, Z{\"u}rich}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{M.Sc. Thesis}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/hst_logo-480.webp 480w,/assets/img/publication_preview/hst_logo-800.webp 800w,/assets/img/publication_preview/hst_logo-1400.webp 1400w," sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/hst_logo.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="hst_logo.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Ravi_btech" class="col-sm-8"> <div class="title">An Induced CD44v6 – EGFR scaffold kinase interaction predisposes a subpopulation of chemotherapy tolerant breast cancer cells and serves as a bi-valent target of resistance</div> <div class="author"> <em>Sudharshan Ravi</em> </div> <div class="periodical"> <em>Harvard - MIT HST</em> , 2012 </div> <div class="periodical"> B.Tech Thesis </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/theses/bachelor_thesis.pdf" target="_blank" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>The failure of chemotherapeutic agents to completely ablate tumor cells and the resulting relapse remain key obstacles of effective cancer therapy. Despite the initial success of most cancer treatments, failure over time is accompanied by the acquisition of chemoresistance. A subpopulation of cancer cells is inherently tolerant to chemotherapy. These cells exert a non-mutational and deterministic alteration to confer a survival advantage during chemotherapy treatment. Although P13K/Akt signaling pathway does help cancer cells to survive lethal doses of chemotherapy agents, elucidating the mechanism to target these cells is required. In-vitro and in-vivo models revealed that the transmembrane glycoprotein, CD44v6 is involved in recruiting a complex scaffold with Ezrin/Radixin/Moesin (ERM) and Akt. Epidermal growth factor receptor (EGFR) activates this cell cortex scaffold in a kinase dependent mechanism along with physical interaction with CD44v6 which renders a subpopulation of chemotherapy tolerant cells sensitive to treatment. A novel bi-specific targeting of CD44v6-EGFR scaffold kinase interaction leads a innovative treatment for relapse in breast cancer.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@thesis</span><span class="p">{</span><span class="nl">Ravi_btech</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ravi, Sudharshan}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{An Induced CD44v6 – EGFR scaffold kinase interaction predisposes a subpopulation of chemotherapy tolerant breast cancer cells and serves as a bi-valent target of resistance}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{Harvard - MIT HST}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2012}</span><span class="p">,</span>
  <span class="na">note</span> <span class="p">=</span> <span class="s">{B.Tech Thesis}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </div> </article> <br> </div> </div> <footer class="sticky-bottom mt-5" role="contentinfo"> <div class="container"> © 2024 Sudharshan Ravi, Ph.D.. <br> <a href="/contact/">Impressum</a>. Last updated: April 08, 2024. <br>Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. <br>Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?da39b660470d1ba6e6b8bf5f37070b6e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>