<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> ΔFBA | Sudharshan Ravi, Ph.D. </title> <meta name="author" content="Sudharshan Ravi, Ph.D."> <meta name="description" content="A workflow for the integration of differential transcriptomics data in genome scale metabolic models"> <meta name="keywords" content="computational biologist, metabolic network analysis, bioinformatics, systems biology"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/Logo2.png?03f154e9931b9a76e3a320234b809c84"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ravisudharsh.github.io/research/deltafba/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?757986ae90a8b8966263cb9d9a515daa"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Sudharshan</span> Ravi, Ph.D. </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item active"> <a class="nav-link" href="/research/">Research <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">Publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/supervision/">Supervision </a> </li> <li class="nav-item "> <a class="nav-link" href="/assets/pdf/CV_SRavi.pdf" target="_blank">Vitae </a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">Contact </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">ΔFBA</h1> <p class="post-description">A workflow for the integration of differential transcriptomics data in genome scale metabolic models</p> </header> <article> <div class="clearfix" style="text-align:justify"> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/r_dfba_main-480.webp 480w,/assets/img/r_dfba_main-800.webp 800w,/assets/img/r_dfba_main-1400.webp 1400w," sizes="95vw" type="image/webp"></source> <img src="/assets/img/r_dfba_main.png" class="img-fluid rounded z-depth-1" width="100%" height="auto" title="DeltaFBA Workflow" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <hr style="height:5px; visibility:hidden;"> <p>Metabolic alterations are often used as hallmarks of observable phenotypes. In this regard, reconstructed genome-scale metabolic models (GEMs) provide a rich and computable representation of the entire set of biochemical reactions in a cell. They provide a powerful framework for simulating the entire set of biochemical reactions in a cell using a constraint-based modeling strategy called flux balance analysis (FBA). FBA relies on an assumed metabolic objective for generating metabolic fluxes using GEMs. But, the most appropriate metabolic objective is not always obvious for a given condition and is likely context-specific, which often complicate the estimation of metabolic flux alterations between conditions. We delevoped a computational tool called ΔFBA (deltaFBA) <a class="citation" href="#Ravi2021">(Ravi &amp; Gunawan, 2021)</a>, that integrates differential gene expression data to evaluate directly metabolic flux differences between two conditions. Notably, ΔFBA does not require specifying the cellular objective. Rather, ΔFBA seeks to maximize the consistency and minimize inconsistency between the predicted flux differences and differential gene expression.</p> <hr style="height:5px; visibility:hidden;"> <p>∆FBA is built in <a href="https://ch.mathworks.com/products/matlab.html" rel="external nofollow noopener" target="_blank">MATLAB</a> with <a href="https://opencobra.github.io/cobratoolbox/stable/index.html" rel="external nofollow noopener" target="_blank">COBRA toolbox</a> compliant functions and subroutines. The tool is freely available on the <a href="https://github.com/CABSEL/DeltaFBA" rel="external nofollow noopener" target="_blank">GitHub page</a> of <a href="https://engineering.buffalo.edu/chemical-biological/people/faculty-directory/core.host.html/content/shared/engineering/chemical-biological/profiles/faculty/gunawan-rudiyanto.detail.html" rel="external nofollow noopener" target="_blank">CABSEL</a>.</p> </div> </article> <br> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/pcbi-480.webp 480w,/assets/img/publication_preview/pcbi-800.webp 800w,/assets/img/publication_preview/pcbi-1400.webp 1400w," sizes="200px" type="image/webp"> <img src="/assets/img/publication_preview/pcbi.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="pcbi.png" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div id="Ravi2021" class="col-sm-8"> <div class="title">ΔFBA—Predicting metabolic flux alterations using genome-scale metabolic models and differential transcriptomic data</div> <div class="author"> <em>Sudharshan Ravi</em>, and Rudiyanto Gunawan </div> <div class="periodical"> <em>PLOS Computational Biology</em>, Nov 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/pubs/journal.pcbi.pdf" target="_blank" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009589" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1371/JOURNAL.PCBI.1009589" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=OW7-mSgAAAAJ&amp;citation_for_view=OW7-mSgAAAAJ:qjMakFHDy7sC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-25-4285F4?logo=googlescholar&amp;labelColor=beige" alt="25 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Genome-scale metabolic models (GEMs) provide a powerful framework for simulating the entire set of biochemical reactions in a cell using a constraint-based modeling strategy called flux balance analysis (FBA). FBA relies on an assumed metabolic objective for generating metabolic fluxes using GEMs. But, the most appropriate metabolic objective is not always obvious for a given condition and is likely context-specific, which often complicate the estimation of metabolic flux alterations between conditions. Here, we propose a new method, called ΔFBA (deltaFBA), that integrates differential gene expression data to evaluate directly metabolic flux differences between two conditions. Notably, ΔFBA does not require specifying the cellular objective. Rather, ΔFBA seeks to maximize the consistency and minimize inconsistency between the predicted flux differences and differential gene expression. We showcased the performance of ΔFBA through several case studies involving the prediction of metabolic alterations caused by genetic and environmental perturbations in Escherichia coli and caused by Type-2 diabetes in human muscle. Importantly, in comparison to existing methods, ΔFBA gives a more accurate prediction of flux differences.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Ravi2021</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ravi, Sudharshan and Gunawan, Rudiyanto}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/JOURNAL.PCBI.1009589}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{1111111111}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1553-7358}</span><span class="p">,</span>
  <span class="na">issue</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Computational Biology}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{Enzyme metabolism,Enzymes,Escherichia coli,Gene expression,Genetics,Metabolites,Soil perturbation,Transcriptome analysis}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">nov</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1009589}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{34758020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{ΔFBA—Predicting metabolic flux alterations using genome-scale metabolic models and differential transcriptomic data}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009589}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">google_scholar_id</span> <span class="p">=</span> <span class="s">{qjMakFHDy7sC}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </div> </div> <footer class="sticky-bottom mt-5" role="contentinfo"> <div class="container"> © 2024 Sudharshan Ravi, Ph.D.. <br> <a href="/contact/">Impressum</a>. Last updated: August 19, 2024. <br>Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. <br>Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?da39b660470d1ba6e6b8bf5f37070b6e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>